docs/yarn.lock

```diff
@@ -1569,18 +1569,18 @@
     react-helmet-async "*"
     react-loadable "npm:@docusaurus/react-loadable@5.5.2"
 
-"@docusaurus/plugin-content-blog@2.1.0":
-  version "2.1.0"
-  resolved "https://registry.yarnpkg.com/@docusaurus/plugin-content-blog/-/plugin-content-blog-2.1.0.tgz#32b1a7cd4b0026f4a76fce4edc5cfdd0edb1ec42"
-  integrity sha512-xEp6jlu92HMNUmyRBEeJ4mCW1s77aAEQO4Keez94cUY/Ap7G/r0Awa6xSLff7HL0Fjg8KK1bEbDy7q9voIavdg==
+"@docusaurus/plugin-content-blog@2.0.1":
```

Comment:
looks like we are downgrading?

Reply:
I think this is related to the --frozen-lockfile update?

Reply:
did you run `yarn` again in `docs` folder? it should update these to 2.1.0 i think?

Reply:
accidentally ran it outside of docs! should be fixed now